src/backup_camera.cpp

```diff
@@ -146,14 +152,32 @@ void BackupCamera::processGPIO()
 void BackupCamera::signalToQuit()
 {
     camera_one_->signalToQuit();
-    //song_player_one_->songQuit();
 }
 
 void BackupCamera::close()
 {
-    //song_player_one_->closeSongPlayer();
-    //song_player_one_->WaitForThreadToExit();
     camera_one_->WaitForThreadToExit();
 }
 
+void BackupCamera::resizeCameraRect(SDL_Window* window)
```

Comment:
Since this function will always be toggling fullscreen, maybe that name would be better. This name implies that you can resize it without toggling fullscreen.